client/src/utils/storage.js

```diff
@@ -0,0 +1,8 @@
+export const saveDataToLocalStorage = data =>
```

Comment:
the names shouldn't be `localStorage` .. the whole idea of abstracting this is to **hide** the implementation detail (the fact that it saves to localStorage), so that we can change the implementation without changing the calls. So it should be `storeUserData` for example ... `item` and `localStorage` are implementation details, we don't want to expose them to external world

Reply:
changed them